src/crypto/crypto.js

```diff
@@ -56,7 +56,7 @@ export function constructParams(types, data) {
  * Encrypts data using specified algorithm and public key parameters.
  * See {@link https://tools.ietf.org/html/rfc4880#section-9.1|RFC 4880 9.1} for public key algorithms.
  * @param {module:enums.publicKey}    algo        Public key algorithm
- * @param {Object}                    pubParams  Algorithm-specific public key parameters
+ * @param {Object}                    publicParams  Algorithm-specific public key parameters
```

Comment:
This is very minor, but since that's the name of this PR: please update the spacing of the surrounding lines so that they line up (even though I know it was already kind of crooked :see_no_evil:). And similarly in the next function :blush:

Reply:
I lined up the jsdoc in the whole file